src/AutoMapper/Bundle/Configuration/RestrictConfigurationPass.php

```diff
@@ -4,6 +4,9 @@
 
 use Jane\AutoMapper\MapperGeneratorMetadataInterface;
 
+/**
+ * @deprecated
```

Comment:
Please add a message with what replaced this interface and on which version this interface will be dropped (v7.x I suppose).

And we should add a class with the new interface to replace that one as well.

Reply:
First part done, second one linked to [our other discussion](https://github.com/janephp/janephp/pull/453#discussion_r527691278)